conans/test/functional/generators/components/test_components_cmake_generators.py

```diff
@@ -286,17 +289,22 @@ def test_standard_names(setup_client_with_greetings, generator):
 
 @pytest.mark.parametrize("generator", ["cmake_find_package_multi", "cmake_find_package"])
 def test_custom_names(setup_client_with_greetings, generator):
+    # NOTE: cpp_info.names is deprecated for CMakeDeps and doesn't work anymore so this test doesn't
```

Comment:
What does this mean? Are things like `self.cpp_info.names["CMakeDeps"] = "MyChat"` not allowed?

Reply:
I wrote that and later at night I thought... it doesn't make sense. It should work, probably some bug. I'm checking